src/styles.scss

```diff
@@ -1 +1,6 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+// Ваші власні SCSS стилі можуть йти після директив Tailwind
```

Comment:
Let's remove these comment. From now write write all future comments only in English